code/modules/mob/mob.dm

```diff
@@ -1061,6 +1061,10 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 	if(M.layer < layer)
 		M.layer = layer + 0.1
 
+///unbuckle the mob from whatever it is buckled to.
+/mob/proc/unbuckle(force)
```

Comment:
Only living mobs are supposed to be bucked, any chance you can change the proc to this?
```suggestion
/mob/living/proc/unbuckle(force)
```

Reply:
I've gone ahead and moved `var/buckled` up to `/mob/living/` entirely.